### PR TITLE
doc: add a doc about how to install golang supernode

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnop@163.com>

### CODE REVIEW

File: docs/user_guide/install_server_go.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    ```sh
+    docker image ls|grep 'supernode' |awk '{print $3}' | head -n1
+    ```
+
+Or you can get it from [DockerHub](https://hub.docker.com/) directly.
+
+1. Obtain the latest Docker image ID of the SuperNode.
+
+    ```sh
+    docker pull supernode:0.4.0
```
Comment (lowzj) [inline docs/user_guide/install_server_go.md]: `supernode:0.4.0` -> `dragonflyoss/supernode:0.4.0`

Comment (Starnop) [reply docs/user_guide/install_server_go.md]: Done.

File: docs/user_guide/install_server_go.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    docker image ls|grep 'supernode' |awk '{print $3}' | head -n1
+    ```
+
+Or you can get it from [DockerHub](https://hub.docker.com/) directly.
+
+1. Obtain the latest Docker image ID of the SuperNode.
+
+    ```sh
+    docker pull dragonflyoss/supernode:0.4.0
+    ```
```
Comment (lowzj) [inline docs/user_guide/install_server_go.md]: move this way to the top of this section

Comment (Starnop) [reply docs/user_guide/install_server_go.md]: Done.

**Pull request merged**